iroh/src/lib.rs

```diff
@@ -1,5 +1,79 @@
 //! Send data over the internet.
 //!
+//! # Getting started
+//!
+//! ## Example
+//!
+//! Create a new node and add some data to the blobs store. This data will be
+//! available over the network.
+//!
+//! ```rust
+//! # async fn run() -> anyhow::Result<()> {
+//! let node = iroh::node::Node::memory().spawn().await?;
+//! let client = node.client();
```

Comment:
I want to keep it in to make it more clear that the client is a separate entity. Otherwise the explanation section below would read weirdly. Client? what client?